### PR TITLE
Add mergePayees method to the API

### PR DESCRIPTION
**Pull request by matt-fidd**
I'm using this for a duplicate payee cleanup script.

Will add to the docs if approved

### CODE REVIEW

**Review by youngcw — APPROVED** [review]

File: packages/loot-core/src/types/api-handlers.d.ts [hunk 1]
```diff
@@ -151,6 +151,11 @@ export interface ApiHandlers {
 
   'api/payee-delete': (arg: { id }) => Promise<unknown>;
 
+  'api/payees-merge': (arg: {
+    targetId: string;
+    mergeIds: string[];
+  }) => Promise<unknown>;
```
Comment (MatissJanis) [inline packages/loot-core/src/types/api-handlers.d.ts:157]: **💬 suggestion:** ‏would you mind adding a proper type here?
```suggestion
  }) => Promise<Something>;
```

File: upcoming-release-notes/3028.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+category: Enhancements
+authors: [matt-fidd]
+---
+
+Add mergePayees method to the API
```
Comment (MatissJanis) [inline upcoming-release-notes/3028.md:6]: **🥜 nitpick:** ‏spellchecker will later complain about this if we don't wrap it in code block
```suggestion
Add `mergePayees` method to the API
```

**Review by MatissJanis — CHANGES_REQUESTED** [review]

File: packages/loot-core/src/types/api-handlers.d.ts (hunk 1 above)
Comment (matt-fidd) [reply packages/loot-core/src/types/api-handlers.d.ts:157]: I'm not sure what type it returns, it seems to pass the return from the database back but when I log out the response from the API call it's always "undefined"

Comment (MatissJanis) [reply packages/loot-core/src/types/api-handlers.d.ts:157]: If it's returning undefined then we can change it to `Promise<void>`

**Review by MatissJanis — APPROVED** [review]
Sweet!

File: packages/loot-core/src/types/api-handlers.d.ts (hunk 1 above)
Comment (matt-fidd) [reply packages/loot-core/src/types/api-handlers.d.ts]: Ah yes, just found it. Updated

https://github.com/actualbudget/actual/blob/6dd34b0c63b54c272c57063ceca7f43f39672df8/packages/loot-core/src/server/db/index.ts#L493-L528

**Pull request merged**